build.gradle

```diff
@@ -175,7 +175,8 @@ project(':ambry-server') {
                 project(':ambry-commons'),
                 project(':ambry-store'),
                 project(':ambry-utils'),
-                project(':ambry-replication')
+                project(':ambry-replication'),
+                project(':ambry-cloud')
```

Comment:
I feel It's better to reverse the dependency and move `VCRServer()` to ambry-cloud. 
All VCR related things are in ambry-cloud.